packages/fxa-settings/src/pages/Pair/AuthAllow/index.tsx

```diff
@@ -0,0 +1,105 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { ReactElement } from 'react';
+import { Link, RouteComponentProps } from '@reach/router';
+import { RemoteMetadata } from '../../../lib/types';
+import { usePageViewEvent } from '../../../lib/metrics';
+import Banner, { BannerType } from '../../../components/Banner';
+import AppLayout from '../../../components/AppLayout';
+import { REACT_ENTRYPOINT } from '../../../constants';
+import DeviceInfoBlock from '../../../components/DeviceInfoBlock';
+import { FtlMsg } from 'fxa-react/lib/utils';
+import { ReactComponent as LocationBalloonImage } from './confirm-pairing.svg';
+
+export type BannerMessage = {
+  messageType: BannerType;
+  messageElement: ReactElement;
+};
+
+export type AuthAllowProps = {
+  authDeviceInfo: RemoteMetadata;
+  // TODO: In FXA-6639 - Listen to broken for error/success messages
+  // included in props temporarily for tests/storybook
+  bannerMessage?: BannerMessage;
+  email: string;
+};
+
+export const viewName = 'pair.auth.allow';
+
+const handleSubmit = () => {
+  // TODO: handle pairing authorization and catch errors in FXA-6639
+};
+
+const AuthAllow = ({
+  authDeviceInfo,
+  bannerMessage,
+  email,
+}: AuthAllowProps & RouteComponentProps) => {
+  usePageViewEvent(viewName, REACT_ENTRYPOINT);
+  const {
+    deviceName,
+    browserName,
+    genericOSName,
+    ipAddress,
+    city,
+    region,
+    country,
+  } = authDeviceInfo;
+
+  const changePasswordLink = (
+    <Link to="/reset_password" className="link-blue">
+      change your password
+    </Link>
+  );
+
+  return (
+    <AppLayout>
+      {/* Does not use CardHeader due to complication of passing vars */}
+      <FtlMsg id="pair-auth-allow-heading-text">
+        <h1 className="card-header">Did you just sign in to Firefox?</h1>
+      </FtlMsg>
+      <p className="card-subheader mb-2">{email}</p>
+      {bannerMessage && (
+        <Banner type={bannerMessage.messageType}>
+          {bannerMessage.messageElement}
```

Comment:
Not sure yet - it will depend how we're receiving these error messages. We might have to compose something similar to `fxa-settings/src/lib/auth-errors`?